docs/docs/plugins.md

```diff
@@ -40,41 +40,9 @@ Plugins can take options. Note that plugin options will be stringified by Gatsby
 See each plugin page below for more detailed
```

Comment:
just for less redundancy, perhaps: "see the links below for instructions on how to use each plugin"